src/Components/Patient/SampleTestCard.tsx

```diff
@@ -92,7 +92,7 @@ export const SampleTestCard = (props: SampleDetailsProps) => {
               Status{" "}
             </div>
             <div className="mt-1 overflow-x-scroll whitespace-normal break-words text-sm font-medium leading-5">
-              {_.startCase(_.camelCase(itemData.status))}
+              {startCase(camelCase(itemData.status))}
```

Comment:
Why not use css to achieve the same result; 
https://tailwindcss.com/docs/text-transform#transforming-text